fuzzbench/jobs.py

```diff
@@ -11,25 +11,42 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Fake jobs."""
+"""Jobs."""
 
+import os
 import subprocess
 import time
 
+BASE_TAG = 'gcr.io/fuzzbench'
 
-def build_image(name):
-    """Build a Docker image."""
+
+def build_base_images(name: str):
+    """Builds a Docker image."""
+    image_tag = os.path.join(BASE_TAG, name)
+    subprocess.run(['docker', 'pull', image_tag], check=True)
+    return subprocess.run(
```

Comment:
this returns a `CompletedProcess` object? is that what we want?

do we actually need to / want to return any value? it might be fine not to return anything (for now) as due to the `check=True` if the function returns that already means "success". wdyt?

Reply:
> this returns a `CompletedProcess` object? is that what we want?

Yes this returns a `CompletedProcess` object. I'm not sure whether this is what we want, but this way we keep information.

> do we actually need to / want to return any value?

That is a good question. I think we should at least return this is success or failed for a run.

> it might be fine not to return anything (for now) as due to the `check=True` if the function returns that already means "success". wdyt?

check=True may need to remove in the future and that is not in default for every job.
For now, I agree, because for build jobs we should expect success.

Let's just return True here. And if we need to add other information in the returned value, we change it.